src/token_enum.h

```diff
@@ -217,6 +217,7 @@ enum c_token_t
    CT_FUNC_TYPE,        // function type - foo in "typedef void (*foo)(void)"
    CT_FUNC_VAR,         // foo and parent type of first parens in "void (*foo)(void)"
    CT_FUNC_PROTO,       // function prototype
+   CT_FUNC_START,       // global DC member for functions(void ::func())
```

Comment:
The token name seems not to be related with resolution operator at all. Is that intentional? Do you have other intents to extend it?

Reply:
Right now i don't have scenarios but for future purposes not included DC member in token name.